pkg/kubeinteraction/status/task_status.go

```diff
@@ -113,8 +116,11 @@ func CollectFailedTasksLogSnippet(ctx context.Context, cs *params.Run, kinteract
 					if strings.HasSuffix(trimmed, " Skipping step because a previous step failed") {
 						continue
 					}
-					// see if a pattern match from errRe
-					ti.LogSnippet = strings.TrimSpace(trimmed)
+					if len(trimmed) > maxErrorSnippetCharacterLimit && utf8.RuneCountInString(trimmed) > maxErrorSnippetCharacterLimit {
+						runes := []rune(trimmed)
+						trimmed = string(runes[:maxErrorSnippetCharacterLimit])
```

Comment:
Since Github's "character limit" is talking about bytes, this will need to be adjusted 😞 

The below suggestion "works" so long as the trim-index doesn't cross any utf8 boundaries.

```suggestion
						trimmed = trimmed[:maxErrorSnippetCharacterLimit]
```

This is a bit ugly but something like it will better handle utf8 boundaries:
```golang
// Trim to max bytes
trimmed = trimmed[:maxErrorSnippetCharacterLimit]

// Trim further to last valid run boundary 
r, size := utf8.DecodeLastRuneInString(trimmed)
for r == utf8.RuneError && size > 0 {
	trimmed = trimmed[:len(trimmed)-size]
	r, size = utf8.DecodeLastRuneInString(trimmed)
}
ti.LogSnippet = trimmed
```